integration/common/constants.go

```diff
@@ -0,0 +1,51 @@
+package common
+
+import (
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/obscuronet/go-obscuro/go/wallet"
+)
+
+// The Contract addresses are the result of the deploying a smart contract from hardcoded owners.
+// The "owners" are keys which are the de-facto "admins" of those erc20s and are able to transfer or mint tokens.
+// The contracts and addresses cannot be random for now, because there is hardcoded logic in the core
+// to generate synthetic "transfer" transactions for each erc20 deposit on ethereum
+// and these transactions need to be signed. Which means the platform needs to "own" ERC20s.
+
+// ERC20 - the supported ERC20 tokens. A list of made-up tokens used for testing.
+// Todo - this will be removed together will all the keys and addresses.
+type ERC20 string
+
+const (
+	HOC            ERC20 = "HOC"
+	POC            ERC20 = "POC"
+	HOCAddr              = "f3a8bd422097bFdd9B3519Eaeb533393a1c561aC"
```

Comment:
minor thing:
HOCAddr is capitalised, while pocAddr not

Reply:
HOC is used in other places while poc isn't, thus the capitalisation difference.